Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/overview/javascript.svelte.mdx

> ## Uploading Source Maps For A Svelte Project

Reviewer: ```suggestion
## Upload Source Maps for a Svelte Project
```

This page is a guide on how to create releases and upload source maps to Sentry when bundling your Svelte app.

To generate source maps with your Svelte project, you need to set the source map [compiler options](https://svelte.dev/docs#compile-time-svelte-compile) in your Svelte config:

```JavaScript {filename:svelte.config.js}
import sveltePreprocess from "svelte-preprocess";

const config = {
  compilerOptions: {
    enableSourcemap: true,
  },
  preprocess: sveltePreprocess({
    sourceMap: true,
  }),
};

export default config;
```

If you're using Vite in you Svelte project, you can use Sentry's Vite plugin for convenience:

```npm
npm install @sentry/vite-plugin --save-dev
```

```yarn
yarn add @sentry/vite-plugin --dev
```

Configure Vite to emit source maps and use the Sentry Vite plugin:

```JavaScript {filename:vite.config.js}
import { defineConfig } from "vite";
import { svelte } from "@sveltejs/vite-plugin-svelte";
import sentryVitePlugin from "@sentry/vite-plugin";

export default defineConfig({
  build: {
    sourcemap: true, // Source map generation must be turned on
  },
  plugins: [
    svelte(),

>     // Put the Sentry esbuild plugin after any other plugins

Reviewer 2: ```suggestion
    // Put the Sentry vite plugin after all other plugins, to ensure no changes will be made to files after they're uploaded
```

    sentryVitePlugin({
      org: "___ORG_SLUG___",
      project: "___PROJECT_SLUG___",

      // Specify the directory containing build artifacts
      include: "./dist",

      // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/

>       // and needs the `project:releases` and `org:read` scopes

Reviewer 2: ```suggestion
      // and need `project:releases` and `org:read` scopes
```

      authToken: process.env.SENTRY_AUTH_TOKEN,

      // Optionally uncomment the line below to override automatic release name detection
      // release: process.env.RELEASE,
    }),
  ],
});
```

### Other Bundlers

> If you're using bundlers other than Vite, check out our general guide on how to <PlatformLink to="/sourcemaps/uploading/">upload source maps</PlatformLink> with them, or refer to their documentation.

Reviewer 2: ```suggestion
If you're using a bundler other than Vite, check out our general guide on how to <PlatformLink to="/sourcemaps/uploading/">upload source maps</PlatformLink>, or refer to your bundler's documentation.
```
